docs/whats-new.md

```diff
@@ -1,11 +1,28 @@
 # What's New
 
-Modules in `@deck.gl-community` are independently maintained, so this page will only list occasional major changes.
+Modules in `@deck.gl-community` are independently maintained and released. 
+The main driver for bigger synchronized releases is to synchronize with 
+deck.gl version upgrades.
 
-Please refer the documentation of each module for detailed news.
+## v9.3 - Currently in planning
 
+Wishlist
+- **`@deck.gl-community/editable-layers`** 
+  - Goal: allow editable layers to be used with non-geospatial coordinates
+  - Goal: make this a generic layer pack (non-geo-speficic).
+- **`@deck.gl-community/graph-layers`**
+  - Goal: allow GraphLayers to be instantiated using `data: URLs` props (rather than requiring applications to prepare GraphEngines).
+- **`@deck.gl-community/infovis-layers`**
+  - Goal: Some of the improved support for (non-geospatial) views etc will be upstreamed into deck.gl v9.3.
```

Comment:
It's a bit odd to include these here, since plans change and they can't be discussed from here.

Could they go in an issue tracker instead?